packages/components/src/components/input/index.ts

```diff
@@ -0,0 +1,13 @@
+import Input from './input.component';
+import { TAG_NAME } from './input.constants';
+import '../button';
```

Comment:
Please add these two component imports as well.
```suggestion
import '../button';
import '../icon';
import '../text';
```